### ISSUE TITLE
probability after decision node causes confusion

### ISSUE THREAD

**Issue by marekantosiewicz** [SilverDecisions/SilverDecisions#205]
It would be better if the probability after a decision node was not shown. from our experience during class it causes unnecessary confusion among students

**Comment by bkamins** [comment]
@mwasiluk - indeed the probability there is either 0.0 or 1.0 - right? Maybe it would be better to show some description rather than a number? @pszufe - what do you think?
This is a low priority thing.

**Comment by pszufe** [comment]
Empty fields should result in error. How does the error look like? I understand it is nicely handled by application (that is I assume this is not the type of "Uknown Exception Errror") , right?

**Comment by bkamins** [comment]
@pszufe - you probably meant this comment in #204?

**Comment by pszufe** [comment]
We could have an option for that that defaults to "turned off".
@mwasiluk are there any scenarios where it is not 0/1 but something in between?

Regardless of that I understand that now this value is accessible via the ${} syntax so I prefer to have numbers here rather than anything else.

**Comment by pszufe** [comment]
> @pszufe - you probably meant this comment in #204?

Sorry, I do not know how I click it but somehow my comments end up in opposite issues. Hopefully I cleared it ;)

**Comment by mwasiluk** [comment]
Maybe we could add an option to display something else for values of probability equal to 0 or 1 and display number if probability is 0 < p < 1 ?

**Comment by bkamins** [comment]
But this is a comment only for decision nodes. I thought in decision nodes we only display 1 (optimal path) or 0 (not optimal path)

**Comment by mwasiluk** [comment]
@bkamins Yes, we display there only 0 and 1, but we should also support displaying (0,1) just in case.

**Comment by bkamins** [comment]
Then let us just leave as it is now. @pszufe - can you make sure that we explain this feature in the book in your chapter. Thank you.
